docs/src/modules/ROOT/pages/quickstart/school-timetabling/school-timetabling-constraints.adoc

```diff
@@ -52,16 +57,61 @@ public class TimetableEasyScoreCalculator implements EasyScoreCalculator<Timetab
 
 }
 ----
+--
+
+Kotlin::
++
+--
+[source,kotlin]
+----
+class TimetableEasyScoreCalculator : EasyScoreCalculator<Timetable?, HardSoftScore?> {
```

Comment:
both Timetable and HardSoftScore should be not-null (no "?"), because the interface's only method will never be called with a null parameter and the returned score should never be null, right?